metrics/seata-metrics-api/src/main/java/org/apache/seata/metrics/IdConstants.java

```diff
@@ -29,6 +29,10 @@ public interface IdConstants {
 
     String APP_ID_KEY = "applicationId";
 
+    String TRANSACTION_ID_KEY = "transactionId";
```

Comment:
ID is a non-enumerable variable and cannot be used as a Prometheus monitoring metric. Otherwise, it will cause Prometheus cache data to inflate infinitely, leading to full GC and ultimately application OOM.